src/test/java/io/vertx/core/http/HttpTest.java

```diff
@@ -5061,6 +5061,49 @@ public void testKeepAliveTTL() throws Exception {
     testKeepAliveTimeout(options, 1);
   }
 
+  @Test
+  public void testKeepAliveTTLWith2Req() throws Exception {
```

Comment:
@vietj this is something you had in mind right?

Reply:
the test needs to check that each connection obtained for the two responses are different instances (i.e not the same physical connections)

Reply:
Added test for same.